src/GameScore/GameScoreDriver.cpp

```diff
@@ -8,10 +8,8 @@
 
 void gameScoreExample() {
     MapLoader loader;
-    std::string fileLocation = "../assets/map1.map";
+    std::string fileLocation = "assets/map1.map";
```

Comment:
```suggestion
    std::string fileLocation = "assets/map1.map"; // in CLion add ../ to start of string
```
Might be worth adding a comment.

Reply:
Definitely, added a separate string so we can just quickly uncomment if using a different IDE. Although I'm finding myself doing more C++ in VSCODE now that I know how to compile it haha.